torchvision/ops/boxes.py

```diff
@@ -29,8 +29,7 @@ def nms(boxes, scores, iou_threshold):
         of the elements that have been kept
         by NMS, sorted in decreasing order of scores
     """
-    _C = _lazy_import()
-    return _C.nms(boxes, scores, iou_threshold)
+    return torch.ops.torchvision.nms(boxes, scores, iou_threshold)
```

Comment:
Can you add a TODO to remove `nms` as an entry in https://github.com/pytorch/vision/blob/93bceaf250fe659f4baa9a1ecf79ece70d14b356/torchvision/csrc/vision.cpp#L10

Reply:
I removed the nms entry and left a comment.
Lmk if you think it is better to keep it with a TODO comment instead.